src/snmalloc/override/malloc.cc

```diff
@@ -40,6 +40,12 @@ extern "C"
   {
     return snmalloc::libc::malloc_usable_size(ptr);
   }
+  SNMALLOC_EXPORT 
+  size_t SNMALLOC_NAME_MANGLE(_msize)( 
+    MALLOC_USABLE_SIZE_QUALIFIER void* ptr) 
+  { 
+    return snmalloc::libc::malloc_usable_size(ptr); 
+  } 
```

Comment:
Thanks for doing this.  If you could put it in a `ifdef` that would be great.  Also clang-format is complaining about the whitespace changes. The below suggestions should fix both.

```suggestion

#ifdef _WIN32
  SNMALLOC_EXPORT
  size_t SNMALLOC_NAME_MANGLE(_msize)(
    MALLOC_USABLE_SIZE_QUALIFIER void* ptr)
  {
    return snmalloc::libc::malloc_usable_size(ptr);
  }
#endif
```